centralbank.py

```diff
@@ -60,3 +72,63 @@ def calculate_inflation(self, current_price, time):
             self.onemonth_CPI = (current_price - self.prices_list[-2])/self.prices_list[-2]
             self.twelvemonth_CPI = (current_price - self.prices_list[-13])/self.prices_list[-13]
             self.actual_inflation = self.twelvemonth_CPI
+
+    def regulate(self, firm_id, firm_cash, firm_var):
+        """Method to regulate firms. Checks if their average cash over the last year can cover their average VaR and to
+        what percent. Based on solvency 2 which has a MCR of 85% and SCR of 99.5%.
+            Accepts:
+                firm_id: Type Integer. Firms unique ID.
+                firm_cash: Type list of decimals. List of cash for last twelve periods.
+                firm_var: Type list of decimals. List of VaR for last twelve periods.
+            Returns:
+                Type String: "Good", "Warning", "LoseControl".
+        This method takes a firms average cash and VaR. If average cash above SCR of 99.5% of VaR then all is well,
+        if cash is between 85% and 99.5% then is issued a warning (limits business heavily), if under 85% then firm is
+        sold."""
+        if firm_id not in self.warnings.keys():
+            self.warnings[firm_id] = 0
+
+        avg_firm_cash = np.mean(firm_cash)
+        avg_var = np.mean(firm_var)
+
+        if avg_firm_cash >= 0.995 * avg_var:
+            self.warnings[firm_id] = 0
+        elif avg_firm_cash >= 0.85 * avg_var:
+            self.warnings[firm_id] += 1
```

Comment:
According to the Comment above, `self.warnings[firm_id]` should always be set to 1 in this case and not further incremented. This would lead to the company being dissolved after 2 periods instead of just restricting its business. Is there a reason to do this?